### ISSUE TITLE
Can't install

### ISSUE THREAD

**Issue by DavidMoritz** [simplehrm/simplehr#6]
I am really interested in trying this out but I can't get it to install. 

First, there is an issue with it wanting to use a version of Ruby that doesn't have "bundle install", so I changed the expected version in .ruby-version.
There are a couple references to "Simple HRM" but all I see is "Karma HRM", so I changed those names.
Next I deleted all the Gemfile.lock files.
Then it wants to use and older version of JSON that I can't seem to get, so I changed it to say that it wanted 2.1.0 instead of 1.8.3.
Just when I thought I had it, I can't seem to get past a dependency on "pg" that I don't know how to resolve. I think it wants version 0.21.0 and I can't get it with certain permissions or something.

I don't know how to move forward, but I am really interested in seeing what this app can do!

**Comment by tachyons** [comment]
Hi, Thanks for your interest. This app isn't production ready at this moment and not updated for last 1 year.

Ruby version used this app is 2.3.1, you can just install gem install bundler to get bundler

Simple HRM was the original name of the project and later renamed to Karma HRM, so there may be some places we forgot to change

You can use deploy to heroku button to see how app looks without actually installing in your local machine (Deploy to heroku button do not support plugins due to some technical issues with hit sub modules

**Comment by DavidMoritz** [comment]
AWESOME! I got it working!

A few steps I had to do:

1: gem install bundler (thank you)
2: comment out the gem for 'pg' and use 'mysql' instead in Gemfile as well as removing the dependency 'plpgsql' in schema file (because I prefer MySQL)
3: go into mysql (or other db) and create the databases and user
4: add the database.yml file configured to use the newly created databases
5: delete the Gemfile.lock
6: download plugins and put them into proper locations (with name changes)
7: rbenv local 2.3.1
8: bundle install
9: rake db:migrate
10: rails server

Badda Bing!

I look forward to checking this thing out! I'll happy make pull requests if I have any improvements. Thanks again for making this!
